lib/ddtrace/metrics.rb

```diff
@@ -49,13 +56,14 @@ def default_statsd_client
     def configure(options = {})
       @statsd = options[:statsd] if options.key?(:statsd)
       self.enabled = options[:enabled] if options.key?(:enabled)
+      @env = options[:env] if options.key?(:env)
     end
 
     def send_stats?
       enabled? && !statsd.nil?
     end
 
-    def count(stat, value = nil, options = nil, &block)
+    def count(stat, value = nil, options = {}, &block)
```

Comment:
The default value for `options` has been changed to `{}` to allow for a much easier merging of options in `metric_options`.

Reply:
That's a fair point, I'll try to implement this in `metric_options` once again.